serverless/deploy/github/distributor/README.md

```diff
@@ -224,7 +224,7 @@ jobs:
       - uses: actions-ecosystem/action-add-labels@v1
         with:
           labels: Automerge
-          number: ${{ steps.pr_metadata.outputs.pr }}
+          number: ${{ steps.find_pr_number.outputs.pr }}
```

Comment:
Thanks for catching this - rather than change this line, though, could you rename the ID on L219 to be `pr_metadata` (this breakage was due to the manual merge I did between this file and the live config on `serverless-test`).

That edit should also fix this, but leaves the door open for other types of metadata to come through this channel too.

Reply:
Sure, done.